test/functional/fixtures/regression/t171129/test.js

```diff
@@ -0,0 +1,3 @@
+it('T171129 - MVCxImageAndDataNavigationDemos/ImageSlider/VideoGallery demo page', function () {
```

Comment:
Why have you cut the ticket name? It's `T171129 - Test running is hung on the MVCxImageAndDataNavigationDemos/ImageSlider/VideoGallery demo page`